<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>北京</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>上海</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>广州</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>深圳</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择你喜欢的一线城市:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>350</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>300</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>300</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
